cotainr/tests/cli/test_build.py

```diff
@@ -103,6 +105,15 @@ def test_already_existing_file(self, answer, monkeypatch):
         build = Build(image_path=image_path, base_image=base_image)
         assert build.base_image == base_image
 
+    def test_specifying_accept_licenses(self):
+        # See also the matching TestAddArguments test below
```

Comment:
maybe add the test description here also. what is it we test for and what we expect. just for person in the future to know what it does.

Reply:
In general, we don't describe the tests. Only when we think a test is a bit tricky to understand, we add some comments about what's going on. This might be a mistake. Is there something in particular, you would like to add here?

Reply:
no. for now just follow the "rule" used elsewhere. This is not something we should spent time on now. Lets get this out of the door.

Reply:
I'll leave as is